Makefile

```diff
@@ -246,8 +247,14 @@ ifeq ($(ARCH),amd64)
 	docker tag $(BUILD_IMAGE):latest-$(ARCH) $(BUILD_IMAGE):latest
 endif
 
-image-test: image fv/Dockerfile.test.amd64 bin/pktgen bin/test-workload bin/test-connection bin/calico-felix-race-amd64 image-wgtool
-	docker build -t $(BUILD_IMAGE)-test:latest-$(ARCH) --build-arg QEMU_IMAGE=$(CALICO_BUILD) --file ./fv/Dockerfile.test.$(ARCH) bin;
+ifeq ($(FV_RACE_DETECTOR_ENABLED),true)
+FV_BINARY=calico-felix-race-amd64
```

Comment:
This seem to be forcing `ARCH=amd64` but the `docker build` command below consider $(ARCH).
Is this expected behavior?

Reply:
@fasaxc, can you follow up with this later please? Since this isn't a regression we're merging this through so that I can get unblocked.

Reply:
Yeah, FVs are amd64 only (previously that was just hard-coded in the dockerfile).